.travis/s2n_travis_build.sh

```diff
@@ -38,25 +38,35 @@ if [[ -n "$GCC_VERSION" ]] && [[ "$GCC_VERSION" != "NONE" ]]; then
     alias gcc=$(which gcc-$GCC_VERSION);
 fi
 
+# Find if the environment has more than 8 cores
+JOBS=8
+if [[ -x "$(command -v nproc)" ]]; then
+    UNITS=$(nproc);
+    if [[ $UNITS -gt $JOBS ]]; then
+        JOBS=$UNITS;
+        echo "Using $JOBS jobs";
```

Comment:
/nit Could we move this echo statement to outside the if statement? This way we can have the log of JOBS count  in the build logs for either of the two cases.